book/zappifest/plugins-manifest-format.md

```diff
@@ -187,6 +191,7 @@ Field Key                   | Description
 **min_zapp_sdk**            | minimum zapp version required for the field, when the plugin is a multi platform, please specify the the platform in the following format: `"min_zapp_sdk": { "ios": "1.0.0", "android": "1.0.0" }`
 **whitelisted_account_ids** | Optional. Array of Applicaster's account_ids as individual strings. To find the relevent account_ids go to `accounts.applicaster.com`. If empty, plugin will be available across all accounts.
 **ui_builder_support**      | Indicates if the plugin should be used in Zapp's UI Builder apps.
+**targets**      | Array type, indicates is this plugin should be used on mobile or tv or both devise targets. Currently the valid options is ["mobile"], ["tv"] or ["mobile", "tv"]
```

Comment:
@yuliroth 
1. I think we should add a few words on the context and where it is used. something like, "Some platforms (like Android and Android TV) enables to develop the same plugin for mobile and TV, this param enables to indicates.......".
2. After we will make it a mandatory field (after rollout day), I think we should mark here that it's mandatory.

@liviur we should note it for "rollout day".

Reply:
@lomudi its not going to be a mandatory field, we will set the value if its empty by the platform